### PR TITLE
Remove ovs duplicated services to set CPUAffinity with systemd .d dirs

### PR DESCRIPTION
**Pull request by mangelajo**
With this change we avoid the need to duplicate and mantain a copy of
the OVS packages while maintaining the CPUAffinity setting.

Related-Issue: https://issues.redhat.com/browse/USHIFT-141
Related-Issue: https://bugzilla.redhat.com/show_bug.cgi?id=2106570#c4

**Which issue(s) this PR addresses**:

Closes #<Issue Number>

### CODE REVIEW

**Comment by mangelajo** [comment]
/hold

I'm testing make iso / install

File: packaging/systemd/ovs-vswitchd-microshift.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[Service]
+CPUAffinity=0
```
Comment (ggiguash) [inline packaging/systemd/ovs-vswitchd-microshift.conf:2]: Add new line?

File: packaging/systemd/ovsdb-server-microshift.conf [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[Service]
+CPUAffinity=0
```
Comment (ggiguash) [inline packaging/systemd/ovsdb-server-microshift.conf:2]: Nit: add new line?

File: packaging/systemd/ovs-vswitchd-microshift.conf [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[Service]
```
Comment (ggiguash) [inline packaging/systemd/ovs-vswitchd-microshift.conf:1]: Let's put microshift- as a prefix in the file name to be consistent with other files.

File: packaging/systemd/ovsdb-server-microshift.conf [hunk 4]
```diff
@@ -0,0 +1,2 @@
+[Service]
```
Comment (ggiguash) [inline packaging/systemd/ovsdb-server-microshift.conf:1]: Let's put microshift- as a prefix in the file name to be consistent with other files.

File: packaging/systemd/ovs-vswitchd-microshift.conf (hunk 1 above)
Comment (mangelajo) [reply packaging/systemd/ovs-vswitchd-microshift.conf]: Done

File: packaging/systemd/ovsdb-server-microshift.conf (hunk 2 above)
Comment (mangelajo) [reply packaging/systemd/ovsdb-server-microshift.conf]: Done.

File: packaging/systemd/ovs-vswitchd-microshift.conf (hunk 3 above)
Comment (mangelajo) [reply packaging/systemd/ovs-vswitchd-microshift.conf]: 👍  done.

File: packaging/systemd/ovsdb-server-microshift.conf (hunk 4 above)
Comment (mangelajo) [reply packaging/systemd/ovsdb-server-microshift.conf]: 👍 done

**Pull request merged**